re.sonny.OhMySVG.yaml

```diff
@@ -0,0 +1,18 @@
+app-id: re.sonny.OhMySVG
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: re.sonny.OhMySVG
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--require-version=1.1.2"
+modules:
+  - name: OhMySVG
+    buildsystem: meson
+    sources: 
+      "type": "git"
+      "url": "https://github.com/sonnyp/OhMySVG.git"
+      "tag": "v1.0.0"
+      "commit": "c3b505ce98aeec0277a8721b2ea5aa5852e913d9"
```

Comment:
```suggestion
      - type: git
        url: https://github.com/sonnyp/OhMySVG.git
        tag: v1.0.0
        commit: c3b505ce98aeec0277a8721b2ea5aa5852e913d9
```

Reply:
thanks

could you give it an other shot?